ledger/acctonline.go

```diff
@@ -656,23 +656,20 @@ func (ao *onlineAccounts) lookupOnlineAccountData(rnd basics.Round, addr basics.
 			// no such online account, return empty
 			return ledgercore.OnlineAccountData{}, err
 		}
-		// lookupOnlineHistory does not fetch the account db round because of the following observation:
-		// 1. ao.onlineAccountsCache update happens with ao.accountsMu taken below and in postCommit
-		// 2. If we started reading the history (lookupOnlineHistory)
-		//   1. before commitRound or while it is running => OK, read what is in DB and then add new entries in postCommit
```

Comment:
I'd like to preserve these cases consideration b/c it explains logic when we need to retry and etc

Reply:
OK, updated